2200026/1225.py

```diff
@@ -0,0 +1,34 @@
+# 암호생성기 D3
+
+for t in range(1, 10):
+    N = int(input())
+    # 8개의 숫자를 입력받는다
+    number = list(map(int, input().split()))
+
+    cnt = 1
+    while True:
+        # 맨 앞의 수에서 cnt를 뺏을때 (num)
+        num = number[0] - cnt
+        # num이 0보다 작거나 같다면
+        if num <= 0:
+            # num을 0으로 셋팅
+            num = 0
+        cnt += 1
+        if cnt == 6:  # cnt가 6이되면 1로 셋팅
+            cnt = 1
+        number.pop(0)  # 첫번째 값 빼고
+        number.append(num)  # 첫번째 값에서 cnt뺀 숫자를 append
+
+        # 주어지는 각 수는 integer 범위를 넘지 않는다.
+        if (number[0] < 10) and (number[1] < 10) and (number[2] < 10) and (number[3] < 10) and (number[4] < 10) and (number[5] < 10) and (number[6] < 10) and (number[7] < 10):
+            break
```

Comment:
거의 다 풀어주셨네요!
```suggestion
        # 주어지는 각 수는 integer 범위를 넘지 않는다.
        if number[7] == 0:
            break
```
비밀번호 생성 종료 시점을 다시 한 번 확인해주세요!